hs_core/views/resource_folder_hierarchy.py

```diff
@@ -71,6 +71,15 @@ def data_store_structure(request):
     res_coll = os.path.join(resource.root_path, store_path)
     try:
         store = istorage.listdir(res_coll)
+        dirs = []
+        for dname in store[0]:  # directories
+            d_pk = dname.decode('utf-8')
+            name_with_full_path = os.path.join(res_coll, d_pk)
+            # TODO /django_irods/download/ shouldn't be hardcoded
+            d_url = '/django_irods/download/zips/' + name_with_full_path
```

Comment:
I got the django_irods path from the url to download files.  Is this the correct way to go?

Reply:
This should work:
```istorage = resource.get_irods_storage()```
```d_url = istorage.url(os.path.join('zips', name_with_full_path))```

Reply:
ty